### PR TITLE
Improve landing page

### PR DESCRIPTION
**Pull request by QuincyLarson**
These are my proposed improvements (simplifications) to our landing page. I've also made some minor improvements to our about page.

### CODE REVIEW

**Comment by atjonathan** [comment]
### **LGTM :+1:**

File: server/views/resources/about.jade [hunk 1]
```diff
@@ -127,35 +127,31 @@ block content
...
-                <h4>Is there a way I can help nonprofits before I earn my certifications?</h4>
-                <p class="negative-15">Yes. You can <a href="/commit">pledge a monthly donation to a nonprofit of your choosing</a> while you learn.</p>
+                <p class="negative-15">Our community has donated more than a million dollars worth of pro-bono coding for nonprofits so far.</p>
+                <h4>Is there a way I can help nonprofits before I earn my certificates?</h4>
+                <p class="negative-15">Yes. You can <a href="/commit">pledge a monthly donation</a> to a nonprofit of your choosing while you learn.</p>
                 <h4>How long does Free Code Camp take?</h4>
-                <p class="negative-15">It takes about 2,080 hours to earn all four of our certifications. This translates into one year of full-time coding. We’re completely self-paced though, so take as long as you need.</p>
+                <p class="negative-15">It takes about 2,080 hours to complete Full Stack Developer certificate. This translates into one year of full-time coding. We’re completely self-paced though, so take as long as you need.</p>
```
Comment (atjonathan) [inline server/views/resources/about.jade]: ~~complete Full Stack Developer certificate.~~
complete **our** Full Stack Developer certificate.

Comment (QuincyLarson) [inline server/views/resources/about.jade]: Thanks! I've pushed a fix.

**Comment by atjonathan** [comment]
Thanks :smile:

**Pull request merged**